lib/extensions.js

```diff
@@ -89,8 +89,9 @@ function getBinaryUrl() {
   return flags['--sass-binary-url'] ||
          package.nodeSassConfig ? package.nodeSassConfig.binaryUrl : null ||
```

Comment:
I think we should once migrate to something like https://github.com/dominictarr/rc or https://github.com/npm/npmconf so I'd love to have some consistent config variable names and camelCase makes it a bit difficult...